openpathsampling/netcdfplus/cache.py

```diff
@@ -374,7 +374,7 @@ class LRUChunkLoadingCache(Cache):
 
     """
 
-    def __init__(self, chunksize=100, max_chunks=100, variable=None):
+    def __init__(self, chunksize=256, max_chunks=8192, variable=None):
```

Comment:
> I think we should name the optimization 1.X

Agreed -- I'm very curious as to what numbers will work best here, and whether we can write helpers to estimate optimal chunk sizes for a given use case (asking information from the user about their machine, and looking at the system to be studied, for example).